### PR TITLE
Release Ruff 0.5.6

### PR DESCRIPTION
**Pull request by MichaReiser**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+
+### Server
+
+- Make server panic hook more error resilient ([#12610](https://github.com/astral-sh/ruff/pull/12610))
+- Use `$/logTrace` for server trace logs in Zed and VS Code ([#12564](https://github.com/astral-sh/ruff/pull/12564))
+- Keep track of deleted cells for reorder change request ([#12575](https://github.com/astral-sh/ruff/pull/12575))
+
+### Configuration
+
+- \[`flake8-implicit-str-concat`\] Always allow explicit multi-line concatenations when implicit are banned ([#12532](https://github.com/astral-sh/ruff/pull/12532))
```
Comment (AlexWaygood) [inline CHANGELOG.md:26]:
```suggestion
- \[`flake8-implicit-str-concat`\] Always allow explicit multi-line concatenations when implicit concatenations are banned ([#12532](https://github.com/astral-sh/ruff/pull/12532))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+
+### Bug fixes
+
+- \[`flake8-async`\] Avoid unused async when context manager includes `TaskGroup` ([#12605](https://github.com/astral-sh/ruff/pull/12605))
```
Comment (AlexWaygood) [inline CHANGELOG.md:30]:
```suggestion
- \[`flake8-async`\] Avoid flagging `asyncio.timeout`s as unused when the context manager includes `asyncio.TaskGroup` ([#12605](https://github.com/astral-sh/ruff/pull/12605))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+- \[`pylint`\] Respect start index in `unnecessary-list-index-lookup` ([#12603](https://github.com/astral-sh/ruff/pull/12603))
+- \[`pyupgrade`\] Avoid recommending no-argument super in `slots=True` dataclasses ([#12530](https://github.com/astral-sh/ruff/pull/12530))
+- \[`pyupgrade`\] Use colon rather than dot formatting for integer-only types ([#12534](https://github.com/astral-sh/ruff/pull/12534))
+- Fix NFKC normalization bug when removing unused imports ([#12571](https://github.com/astral-sh/ruff/pull/12571))
+
+### Documentation
+
+-
+- Recommend client config for `trace` setting in Neovim ([#12562](https://github.com/astral-sh/ruff/pull/12562))
+
```
Comment (AlexWaygood) [inline CHANGELOG.md:46]:
```suggestion
```

**Review by AlexWaygood — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## 0.5.6
+
+Ruff 0.5.6 automatically enables linting and formatting of notebooks in *preview mode*.
+You can opt-out of this behavior by adding `*.ipynb` to the `extend-exclude` setting.
```
Comment (dhruvmanila) [inline CHANGELOG.md:6]: We could provide an example for this?
```toml
[tool.ruff]
extend-exclude = ["*.ipynb"]
```

**Comment by MichaReiser** [comment]
I'm sure pre-commit will yell at me again for changing the markdown

**Pull request merged**